### PR TITLE
Add docs for input types

### PR DESCRIPTION
**Pull request by patrick91**

### CODE REVIEW

File: docs/types/input-types.md [hunk 1]
```diff
@@ -4,4 +4,49 @@ title: Input types
 
 # Input types
 
-Documentation coming soon
+In addition to [object types](./object/types) GraphQL also supports input types. While being similar to object types, they are better suited for input data as they limit the kind of types you can use for fields.
```
Comment (patrick91) [inline docs/types/input-types.md:7]: reminder to self: fix the link here

**Comment by botberry** [comment]
Hi, thanks for contributing to Strawberry 🍓!

We noticed that this PR is missing a `RELEASE.md` file. We use that to automatically do releases here on GitHub and, most importantly, to PyPI!

So as soon as this PR is merged, a release will be made 🚀.

Here's an example of `RELEASE.md`:

```markdown
Release type: patch

Description of the changes, ideally with some examples, if adding a new feature.
```

Release type can be one of patch, minor or major. We use [semver](https://semver.org/),so make sure to pick the appropriate type. If in doubt feel free to ask :)

File: docs/types/input-types.md [hunk 2]
```diff
@@ -4,4 +4,49 @@ title: Input types
...
+@strawberry.type
+class Mutation:
+    @strawberry.mutation
+    def store_point(self, a: Point2D) -> bool:
+        return True
+```
+
+## API
+
+`@strawberry.type(name: str = None, description: str = None)`
```
Comment (AGASS007) [inline docs/types/input-types.md:47]: `@strawberry.input` ?

Comment (patrick91) [reply docs/types/input-types.md:47]: good shout! thanks 😊 fixed in 09d23d9

**Pull request merged**